pkgs/top-level/python-packages.nix

```diff
@@ -2616,17 +2623,29 @@ in modules // {
       py.test $out/${python.sitePackages}
     '';
 
-    # Marked broken since according to test
+    # FIXME: make gdk_pixbuf dependency optional (as wel as xcfffi)
     # Happens with 0.7.1 and 0.7.2
     # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
 
-    patchPhase = ''
+    patches = [
```

Comment:
On Sun, Feb 07, 2016 at 05:32:44AM -0800, Nikolay Amiantov wrote:

> > -    patchPhase = ''
> > +    patches = [
> 
> Usual way before new Nix release was to use _some_ checksum 
> 

Done.
Check please, if `nix-build -A pkgs.pythonPackages.cairocffi` works for you after update, it will work everywhere.

TY for your help.